recipes/ngless/meta.yaml

```diff
@@ -1,33 +1,82 @@
 {% set name = "ngless" %}
 {% set version = "1.1.0" %}
-{% set sha256 = "9706b5538f44f1962f6cd891672b4ff90a5b30523e91613df169e06ef61a8278" %}
+{% set sha256 = "7461d0ab2eacedf7e378f90c195d170758f20f5413fc31337315a35040d5258a" %}
+
+{% set ghc_src_version = "8.6.5" %}
+{% set ghc_src_sha256 = "4d4aa1e96f4001b934ac6193ab09af5d6172f41f5a5d39d8e43393b9aafee361" %}
+{% set ghc_bootstrap_version = "8.4.2" %}
+{% set ghc_bootstrap_linux_sha256 = "28c31489ed9a83af4400685ec8bc72a6d43d08958d75b2dc35d29a894a5c9d93" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/ngless-toolkit/ngless/releases/download/v{{ version }}/NGLess-{{ version }}-static-Linux64
-  sha256: {{ sha256 }}
+  - url: https://github.com/ngless-toolkit/ngless/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    folder: ngless
+  - url: https://downloads.haskell.org/~ghc/{{ ghc_bootstrap_version }}/ghc-{{ ghc_bootstrap_version }}-x86_64-centos67-linux.tar.xz  # [linux]
+    sha256: {{ ghc_bootstrap_linux_sha256 }}  # [linux]
+    folder: ghc_bootstrap
+
+  - url: https://downloads.haskell.org/~ghc/{{ ghc_src_version }}/ghc-{{ ghc_src_version }}-src.tar.xz
+    sha256: {{ ghc_src_sha256 }}
+    folder: ghc_src
+
 
 build:
-  number: 0
-  skip: True  #  [osx]
+  number: 1
+  skip: True # [osx]
+
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - perl
+    - coreutils
+    - binutils
+    - pkg-config
+    - automake
+    - autoconf
+    - libtool
+    - stack >=1.9.3
+  host:
+    - gmp
+    - xz
+    - zlib
+    - bzip2
+    - backports.lzma
+    - ncurses
+    - libmagic
```

Comment:
Some of these dependencies (like libmagic) were for git-annex, prob can be dropped

Reply:
Thanks for the feedback. I'm running some of this locally now. Unfortunately, the testing is very slow on the machine I am using, so it may take a while if I have to do a few iterations. I'll only push a working version, though.